### PR TITLE
Fixes #12087 - host create VMware docs update

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: doc/host_create.md [hunk 1]
```diff
@@ -221,13 +221,18 @@ flavor_id
...
-## vmWare
+## VMware
 Available keys for `--compute-attributes`:
 ```
-cpus       # cpu count
-memory_mb  # integer number
-cluster
-path
+cpus                 # cpu count
+corespersocket       # number of cores per socket
```
Comment (mmoll) [inline doc/host_create.md]: I'd take that out because that might disappear with #10873 anyway

**Comment by mmoll** [comment]
:+1:

**Comment by mmoll** [comment]
what's the status here?

File: doc/host_create.md (hunk 1 above)
Comment (tstrachota) [inline doc/host_create.md]: I added a note on the line below.

**Comment by tstrachota** [comment]
Thanks for pointing it out @mmoll, updated.

**Comment by mbacovsky** [comment]
:+1: Thank you guys :pig:

**Pull request merged**